config/v1/types_scheduling.go

```diff
@@ -25,6 +25,21 @@ type SchedulingSpec struct {
 	// The namespace for this configmap is openshift-config. 
 	// +optional
 	Policy ConfigMapNameReference `json:"policy,omitempty"`
+	// defaultNodeSelector helps set the cluster-wide default node selector to 
+	// restrict pod placement to specific nodes. This is applied to the pods 
+	// created in all namespaces without a specified nodeSelector value.
+	// For example,
+	// defaultNodeSelector: "type=user-node,region=east" would set nodeSelector
+	// field in pod spec to "type=user-node,region=east" to all pods created
+	// in all namespaces. Namespaces having project-wide node selectors won't be 
+	// impacted even if this field is set. This adds an annotation section to 
+	// the namespace. 
+	// For example, if a new namespace is created with 
```

Comment:
is this an example of what annotation to set on the namespace?  if so, perhaps say that if the `openshift.io/node-selector` annotation is set on the namespace, the value is used in preference to this default.  for instance `openshift.io/node-selector: "type=user-node,region=west"` means that the default of `"type=user-node,region=east"` would not be applied

Reply:
Done @deads2k